guide/canvas/README.md

```diff
@@ -0,0 +1,334 @@
+## Setting up Canvas
+
+<p class="tip">For this guide, we will be using `canvas@next`. At the time of writing, that is `canvas@2.0.0-alpha.12`, so ensure your `package.json` has that or a similar version after installation.</p>
+
+Canvas is an image manipulation tool that allows you to basically modify pictures with code. We will explore how to use this module in a heavily requested feature: guild member welcome messages. But first, we must go through the intense labour of installing canvas. I highly reccommend using a linux distro because it is much easier to install.
```

Comment:
```diff
- labour
+ labor
```
![angery](https://cdn.discordapp.com/emojis/407630001052188692.png?v=1)

Reply:
```diff
- I highly reccommend using a linux distro because it is much easier to install.
+ It's highly recommended that you use a Linux ditro for this because it'll be much easier to install on.